Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -564,9 +566,12 @@ - (void)viewWillAppear:(BOOL)animated
     // We need to `beginLongLivedReadTransaction` before we update our
     // mapping in order to jump to the most recent commit.
     [self.uiDatabaseConnection beginLongLivedReadTransaction];
+    self.messageMappings = [[YapDatabaseViewMappings alloc] initWithGroups:@[ self.thread.uniqueId ]
+                                                                      view:TSMessageDatabaseViewExtensionName];
```

Comment:
This isn't free, but I think we have to do this work - the `messageMappings` may have become stale while we were ignoring the modification notifications. See `toggleObservers:YES` below.